cli/run_model.py

```diff
@@ -9,8 +9,9 @@
 _logger = logging.getLogger(__name__)
 
 
-@click.group()
+@click.group('model')
 def main():
+    """Run models"""
```

Comment:
Since you are using click you can just add my modeling as a subcommand directly right now. 

If you have an existing `group` you can use `group.add_command(model)` where model is another group that can be imported from my libs. This allows for the same modeling subcommands to be used as well.

Reply:
yeah, excited for that :)